### ISSUE TITLE
Training on custom data get stuck

### ISSUE THREAD

**Issue by inders** [ildoonet/tf-pose-estimation#303]
I am trying to use trained weights on COCO to train on my custom dataset

[2018-08-23 18:02:29,767] [train] [INFO] Restore pretrained weights...Done
[2018-08-23 18:02:29,767] [train] [INFO] prepare file writer
[2018-08-23 18:02:31,759] [train] [INFO] prepare coordinator
[2018-08-23 18:02:31,760] [train] [INFO] Training Started.
[2018-08-23 18:02:46,416] [train] [INFO] epoch=912.98 step=114122, 1.0918 examples/sec lr=0.000000, loss=4.69139, loss_ll=0.947289, loss_ll_paf=0.748125, loss_ll_heat=1.14645, q=73
0it [00:00, ?it/s]

Any suggestions would help.

**Comment by chuck0518** [comment]
@inders Excuse me,inders， I want to ask if training convergence is related to the number of epochs? I found that you have more than 900 epochs and the loss is very low. My iterations are only 30 epochs, and the loss only drops to around 150. Just because the server has only two gpus, it is very slow to train. Can you give some advice? 

This is my train  parameter：
python tf_pose/train.py --model=mobilenet_thin --datapath='/home/dl/Public-ML/ssd/COCO/' --batchsize=16 --lr=0.001

**Comment by Jay1015** [comment]
@inders Hi inders，Have you solved this problem? If you successfully solve it, can you tell me how to solve it? Thanks!!
